Touch Bar Simulator/main.swift

```diff
@@ -67,6 +68,11 @@ final class AppDelegate: NSObject, NSApplicationDelegate, NSWindowDelegate {
 		controller.window!.alphaValue = CGFloat(sender.doubleValue)
 		UserDefaults.standard.set(sender.doubleValue, forKey: "windowTransparency")
 	}
+    
+	@objc func toggleView(_ pboard: NSPasteboard, userData: String, error: NSErrorPointer) {
+		let a = controller.window!.isVisible
+		controller.window!.setIsVisible(!a)
```

Comment:
Yes, I initially did this, but for an unknown case it didn't work. Let's try again. Maybe it was just something related to cache or other stuffs.